front/src/reducers/editor/thunkActions.ts

```diff
@@ -15,7 +15,6 @@ import type { Operation } from 'common/api/osrdEditoastApi';
 import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import i18n from 'i18n';
 import { setLoading, setSuccess, setFailure, setSuccessWithoutMessage } from 'reducers/main';
-import { updateIssuesSettings } from 'reducers/map';
```

Comment:
Now that updateIssuesSettings is exported from the editor reducer, you should be able to import it like this
```suggestion
import { updateIssuesSettings } from 'reducers/editor';
```

And l.158, you can remove the editorSliceActions you have added